crates/rspack_core/src/cache/persistent/storage/memory_gc.rs

```diff
@@ -0,0 +1,65 @@
+use std::sync::atomic::{AtomicU32, Ordering};
```

Comment:
It is not appropriate to place memory_gc in the persistent cache, storage in persistent cache is used to save buffer data to database/file. memory_gc is more like a util, maybe it would be better to move it to `src/utils`.